docs/source/models.rst

```diff
@@ -904,27 +904,49 @@ The REST API server accepts the following data formats as POST input to the ``/i
 * CSV-serialized pandas DataFrames. For example, ``data = pandas_df.to_csv()``. This format is
   specified using a ``Content-Type`` request header value of ``text/csv``.
 
+* Numpy arrays formatted as described in `TF Serving's API docs
```

Comment:
Hmm, should we call this Tensor input instead of numpy arrays since it can be a dict of numpy arrays? Not sure, which one is more clear.

Reply:
ooh yeah makes sense